docs/architecture/adr-027-deterministic-protobuf-serialization.md

```diff
@@ -12,20 +12,20 @@ Proposed
 ## Abstract
 
 Fully deterministic structure serialization, which works across many languages and clients,
-is needed for structure signature use-case. We need to be sure that whenever we serialize
+is needed for the use case of structure signatures. We need to be sure that whenever we serialize
```

Comment:
What are `structure signatures`. That's really not very clear

Reply:
Maybe `is needed for generating a signature payload.` ?

Reply:
How about _is needed to sign a structure._?

Reply:
My reasoning was, `Fully deterministic structure serialization is needed...` to sign a struct, an array, a string, an integer, any kind of signature payload.

Reply:
Message is the protobuf term.

Reply:
Updated to "needed when signing messages".  I hope that is more clear.